### PR TITLE
Fixing permissions access

### LINKED ISSUE #178: Rename file and other actions are shown when a user has no permission
When a non-admin logs in, it has the option to change permissions, rename a file, etc. for files it has no ownership over.

Some notable examples:

* [ ] create link
* [ ] rename
* [ ] move
* [ ] change permissions

### PR DESCRIPTION
**Pull request by Abdelra6**
This PR aims to fix #178

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -231,25 +231,50 @@ export const Application = () => {
...
-                                : [],
-                            { type: "divider" },
-                            { title: _("Edit properties"), onClick: _editProperties },
-                            { title: _("Rename"), onClick: _renameItem },
-                            { type: "divider" },
-                            { title: _("Create link"), onClick: _createLink },
-                            { type: "divider" },
-                            { title: cockpit.format(_("Delete")), onClick: _deleteItem, className: "pf-m-danger" },
-                        ])
+                        : superuser.allowed
```
Comment (jelly) [inline src/app.jsx:234]: This doesn't work, this needs to deal with UNIX permissions from the selected file. More details for example [here](https://www.redhat.com/sysadmin/linux-file-permissions-explained).

Comment (Abdelra6) [reply src/app.jsx:234]: What exactly doesn't work? Did I misunderstand the issue or implemented it wrongly?
I hid create link while in "limited access" mode

Comment (jelly) [reply src/app.jsx:234]: The code needs to look at the current file permissions and the current logged in user, not if they are `sudo` or not.

I can only recommend researching how UNIX file permissions work in general https://linuxize.com/post/understanding-linux-file-permissions/

Comment (Abdelra6) [reply src/app.jsx:234]: Aha So every file should be worked on independently? but I should hide for example "create link" Like I did ?

Comment (jelly) [reply src/app.jsx:234]: Yes, every file needs to be checked independent.

**Pull request closed without merging**